src/components/views/elements/TagComposer.tsx

```diff
@@ -64,6 +67,17 @@ export default class TagComposer extends React.PureComponent<IProps, IState> {
         this.props.onRemove(tag);
     }
 
+    private onValidateKeyword = async (fieldState: IFieldState) => {
+        if (!this.props.onValidate) {
+            this.setState({ isNewTagValid: true });
+            return { };
+        }
+        const result = await this.props.onValidate(fieldState);
+        this.setState({ isNewTagValid: result.valid });
+
+        return result;
```

Comment:
The blank line seems like it's in a weird spot:
```suggestion

        const result = await this.props.onValidate(fieldState);
        this.setState({ isNewTagValid: result.valid });
        return result;
```